### PR TITLE
Chrome to electron

### PR DESCRIPTION
**Pull request by yavorsky**
Add chromiumToElectron method with reversed logic.
For performance and other purposes also added it to build.js for `chrome-versions.js` and `full-chrome-versions.js` builds.

### CODE REVIEW

**Comment by Kilian** [comment]
Nice extension, code looks fine :) Any particular use for this in mind? 

Could you update the README.md with the new functions and lists? The full chrome versions returns an array instead of a string so that would be important to communicate clearly.

**Comment by yavorsky** [comment]
@Kilian Ah, yes!

**Comment by Kilian** [comment]
Also needs the description for importing the individual version files. I don't mind doing this but don't know when I'll get around to it. If you beat me to it that would be great ;)

File: index.js [hunk 1]
```diff
@@ -1,26 +1,36 @@
...
   if (query === 1) { number = "1.0" }
   if (typeof query === 'number') { number += ''; }
   return number;
-}
+};
 
 module.exports = {
   versions: versions,
   fullVersions: fullVersions,
+  chromeVersions: chromeVersions,
```
Comment (Kilian) [inline index.js]: please use 'chromium' for consistency

**Review by Kilian — CHANGES_REQUESTED** [review]
Please use "chromium" instead of "chrome" for consistency, in the property names and filenames.

Please make sure that the new properties and files are added to the readme (with chromium in their names)

**Comment by Kilian** [comment]
I made the above changes in this branch: https://github.com/Kilian/electron-to-chromium/tree/yavorsky-chrome-to-electron 

@yavorsky did I miss anything? if not, I'll merge that branch.

**Comment by yavorsky** [comment]
@Kilian Can you please open PR `yavorsky-chrome-to-electron` to `master` to compare changes, reviews, etc? 🙂

**Comment by Kilian** [comment]
Closing this to continue in #8.

**Pull request closed without merging**